tests/models/mistral/test_modeling_mistral.py

```diff
@@ -455,10 +456,10 @@ def test_model_7b_logits(self):
     @slow
     def test_model_7b_generation(self):
         EXPECTED_TEXT_COMPLETION = """My favourite condiment is 100% ketchup. I love it on everything. I’m not a big"""
-        prompt = "My favourite condiment is "
+        prompt = "My favourite cnondiment is "
         tokenizer = AutoTokenizer.from_pretrained("mistralai/Mistral-7B-v0.1", use_fast=False)
-        input_ids = tokenizer.encode(prompt, return_tensors="pt").to(torch_device)
         model = MistralForCausalLM.from_pretrained("mistralai/Mistral-7B-v0.1", device_map="auto")
+        input_ids = tokenizer.encode(prompt, return_tensors="pt").to(model.model.embed_tokens.weight.device)
```

Comment:
Using `device_map="auto"` seems put the model on gpu sometimes (maybe part of time) but on cpu another times, and the results are different.

Will have to see if there is a way to make this deterministic and pass the test.